### PR TITLE
added Collections service

### PR DESCRIPTION
**Pull request by mlaggner**
added collection info to the movie service and the whole collection service
added images to the AppendToResponseItem

### CODE REVIEW

File: src/main/java/com/uwetrottmann/tmdb/Tmdb.java [hunk 1]
```diff
@@ -17,15 +17,7 @@
...
-import com.uwetrottmann.tmdb.services.ConfigurationService;
-import com.uwetrottmann.tmdb.services.DiscoverService;
-import com.uwetrottmann.tmdb.services.FindService;
-import com.uwetrottmann.tmdb.services.MoviesService;
-import com.uwetrottmann.tmdb.services.PeopleService;
-import com.uwetrottmann.tmdb.services.SearchService;
-import com.uwetrottmann.tmdb.services.TvEpisodesService;
-import com.uwetrottmann.tmdb.services.TvSeasonsService;
-import com.uwetrottmann.tmdb.services.TvService;
+import com.uwetrottmann.tmdb.services.*;
```
Comment (UweTrottmann) [inline src/main/java/com/uwetrottmann/tmdb/Tmdb.java]: No star imports please.

File: src/main/java/com/uwetrottmann/tmdb/Tmdb.java [hunk 2]
```diff
@@ -166,4 +158,5 @@ public DiscoverService discoverService() {
         return getRestAdapter().create(DiscoverService.class);
     }
 
+    public CollectionService collectionService() { return getRestAdapter().create(CollectionService.class); }
```
Comment (UweTrottmann) [inline src/main/java/com/uwetrottmann/tmdb/Tmdb.java]: No single line methods, please use new lines (see [Google Java Style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.1-braces) for a good starting point).

File: src/main/java/com/uwetrottmann/tmdb/services/CollectionService.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import com.uwetrottmann.tmdb.entities.AppendToResponse;
+import com.uwetrottmann.tmdb.entities.Collection;
+import com.uwetrottmann.tmdb.entities.Images;
+import retrofit.http.GET;
+import retrofit.http.Path;
+import retrofit.http.Query;
+
+public interface CollectionService {
+    @GET("/collection/{id}")
+    Collection summary(@Path("id") int var1, @Query("language") String var2, @Query("append_to_response") AppendToResponse var3);
```
Comment (UweTrottmann) [inline src/main/java/com/uwetrottmann/tmdb/services/CollectionService.java]: Please at least copy the method description from the API docs and add it as javadoc. Also, parameters could use some descriptive names. See the other services for examples.

**Comment by UweTrottmann** [comment]
Thanks very much for the PR!

I added some inline comments about things you should improve to maintain consistency with the existing code. If you don't have the time, I will do the changes for you (let me know) before merging.

**Comment by mlaggner** [comment]
I'll change it in the evening.
2 of the 3 things you mentioned did the IDE without my interaction (still getting used to IntelliJ IDEA.. :/)

**Comment by UweTrottmann** [comment]
No worries :+1: To "turn off" star imports see e.g. [this stackoverflow](http://stackoverflow.com/a/26281255/1000543).

**Comment by mlaggner** [comment]
I've changed the thing you pointed out. Thanks for the hint with the settings.

File: src/test/java/com/uwetrottmann/tmdb/services/MoviesServiceTest.java [hunk 4]
```diff
@@ -3,17 +3,7 @@
...
-import com.uwetrottmann.tmdb.entities.Images;
-import com.uwetrottmann.tmdb.entities.ListResultsPage;
-import com.uwetrottmann.tmdb.entities.Movie;
-import com.uwetrottmann.tmdb.entities.MovieAlternativeTitles;
-import com.uwetrottmann.tmdb.entities.MovieKeywords;
-import com.uwetrottmann.tmdb.entities.MovieResultsPage;
-import com.uwetrottmann.tmdb.entities.Releases;
-import com.uwetrottmann.tmdb.entities.ReviewResultsPage;
-import com.uwetrottmann.tmdb.entities.Videos;
+import com.uwetrottmann.tmdb.entities.*;
```
Comment (UweTrottmann) [inline src/test/java/com/uwetrottmann/tmdb/services/MoviesServiceTest.java]: Still a star import here :)

**Comment by UweTrottmann** [comment]
Great! You missed one star import though.

Once done, it would be nice if you could squash your commits (e.g. see [rebaseandsqua.sh](http://rebaseandsqua.sh/)).

**Comment by mlaggner** [comment]
hope this works. got many errors while while executing rebaseandsqua.sh
I did all steps manually afterwards.

**Comment by UweTrottmann** [comment]
You should do the rebase against the dev branch (e.g. replace `master` with `dev` where appropriate). Sorry I forgot to explicitly mention that.

**Comment by mlaggner** [comment]
sorry, everything I do now ends in an "Everything up to date". 
I am a git newcomer and already spent twice the time for trying to squash and rebase than it took me to enhance the API..

**Comment by UweTrottmann** [comment]
Alright, here is how you should be able to fix it. I'm assuming that you have [added this repository](https://help.github.com/articles/adding-a-remote/) as the git remote `upstream` (e.g. the GitHub apps create those automatically when checking out a fork).

```
# get latest changes from this repository
git fetch upstream
# rebase your dev branch on top of this repos dev branch
git rebase upstream/dev
# now git will likely have problems due to some redundant commits
# repeat the skip command until the rebase is done
git rebase --skip
# to enter a better commit message for your changes do
git commit --amend -m "Add collections service."
# now push to your forks dev branch, forcing to overwrite with the new version
git push origin dev -f
```

**Comment by UweTrottmann** [comment]
LGTM, thanks again!

**Pull request merged**